worldwind/src/commonMain/kotlin/earth/worldwind/util/kgl/Kgl.kt

```diff
@@ -413,7 +413,8 @@ interface Kgl {
     fun bindBuffer(target: Int, buffer: KglBuffer)
     fun bufferData(target: Int, size: Int, sourceData: ShortArray, usage: Int, offset: Int = 0)
     fun bufferData(target: Int, size: Int, sourceData: IntArray, usage: Int, offset: Int = 0)
-    fun bufferData(target: Int, size: Int, sourceData: FloatArray, usage: Int, offset: Int = 0)
+    fun bufferData(target: Int, size: Int, sourceData: FloatArray?, usage: Int, offset: Int = 0)
```

Comment:
Please implement all it on all platforms and add nulable source data to all array types for consystency.

Reply:
done